back-end/helpers/db-helpers.js

```diff
@@ -24,7 +20,38 @@ async function getRowFromDb(queryString) {
   return results.rows[0];
 };
 
+async function getUserByEmail(email) {
+  const query = `SELECT password, id FROM users WHERE email = '${email}'`;
```

Comment:
While it may not be a concern for our project, direct string concatenation or interpolation risks chances of SQL injection. We should pass these as parameters and the postgres query builder will handle most vulnerabilities

https://node-postgres.com/features/queries#Parameterized%20query

Reply:
👍 i will reuse your `queryWithParameters` method after this is merged in.